specification/network/resource-manager/Microsoft.Network/2017-09-01/networkSecurityGroup.json

```diff
@@ -674,8 +674,6 @@
       },
       "required": [
         "protocol",
-        "sourceAddressPrefix",
```

Comment:
Is there are default value for the now optional properties?

Reply:
With new API version user has to specify either `sourceAddressPrefix`, `sourceAddressPrefixes` or `sourceApplicationSecurityGroups` (same for `destinationAddressPrefix`), but there is no way to specify a condition like "you need to pass one of these N properties" in Swagger, so `sourceAddressPrefix` has to be removed from required properties (otherwise the user would be forced to use it even if they want to use the other parameter).